extensions/panache/mongodb-panache/deployment/src/main/java/io/quarkus/mongodb/panache/deployment/PanacheResourceProcessor.java

```diff
@@ -137,7 +139,7 @@ void buildImperative(CombinedIndexBuildItem index,
             BuildProducer<ReflectiveClassBuildItem> reflectiveClass,
             BuildProducer<PropertyMappingClassBuildStep> propertyMappingClass,
             BuildProducer<PanacheEntityClassesBuildItem> entityClasses,
-            List<PanacheMethodCustomizerBuildItem> methodCustomizersBuildItems) {
+            List<PanacheMethodCustomizerBuildItem> methodCustomizersBuildItems) throws BuildException {
```

Comment:
Seems like the exception is not thrown anymore..

Reply:
Removed